frontend/src/lib/Apis.ts

```diff
@@ -344,14 +344,17 @@ export class Apis {
   public static async uploadPipelineVersion(
     versionName: string,
     pipelineId: string,
+    description: string,
     versionData: File,
   ): Promise<ApiPipelineVersion> {
     const fd = new FormData();
     fd.append('uploadfile', versionData, versionData.name);
     return await this._fetchAndParse<ApiPipelineVersion>(
       '/pipelines/upload_version',
       v1beta1Prefix,
-      `name=${encodeURIComponent(versionName)}&pipelineid=${encodeURIComponent(pipelineId)}`,
+      `name=${encodeURIComponent(versionName)}&pipelineid=${encodeURIComponent(
+        pipelineId,
+      )}&description=${encodeURIComponent(description)}`,
```

Comment:
Can description be optional? If user doesn't set description, can we avoid providing this value in upload version call?

Reply:
sure!